operators/rx.go

```diff
@@ -19,6 +20,7 @@ var _ rules.Operator = (*rx)(nil)
 
 func newRX(options rules.OperatorOptions) (rules.Operator, error) {
 	data := options.Arguments
+	data = fmt.Sprintf("(?sm)%s", data)
```

Comment:
```suggestion
        // (?sm) enables multiline mode, see https://stackoverflow.com/a/27680233
        // for more info
	data = fmt.Sprintf("(?sm)%s", data)
```

Reply:
Also, can we concatenate multiple flags? Like `(?sm)(?i)...`. Some CRS regexes uses the case insensitive flag